co.uk.studiopond.spiderbro.yml

```diff
@@ -0,0 +1,25 @@
+app-id: co.uk.studiopond.spiderbro
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: spiderbro_linux
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+modules:
+  - shared-modules/glu/glu-9.json
```

Comment:
You haven't added the shared-modules submodule here https://github.com/flathub/shared-modules